CONTRIBUTING.template.md

```diff
@@ -46,3 +46,17 @@ The Goldstack monorepo is a repository nested in two levels. [workspaces](https:
 - [template-lib](https://github.com/goldstack/goldstack/tree/master/workspaces/templates-lib) - contains packages that are used as dependencies of templates to support the build process and development.
 - [template-management](https://github.com/goldstack/goldstack/tree/master/workspaces/templates-management) - contains utilities for developing and testing templates.
 - [template](https://github.com/goldstack/goldstack/tree/master/workspaces/templates) - contains the blueprint for Goldstack templates
+
+### Developing Goldstack Alongside your Project
+
+When you generate a new project with Goldstack, this project will be linked to [packages published in NPM](https://www.npmjs.com/search?q=keywords:goldstack). The source code for these libraries is defined in [template-lib](https://github.com/goldstack/goldstack/tree/master/workspaces/templates-lib).
+
+If you want to make a change to one of the libraries and test it directly with your project, the easiest way is to combine your project with the Goldstack monorepo. For this, copy your generated project into the folder: `workspaces/generated`
+
+![](https://user-images.githubusercontent.com/1448524/155213397-2b67a16d-fb76-476e-bfcf-314903dcc046.png)
+
```

Comment:
#### Copy your generated project into the folder: `workspaces/generated`

[img]

It is important to place the generated project into the `workspaces/generated` folder since this is ignored by Git in the Goldstack monorepo.


```suggestion
#### Copy your generated project into the folder: `workspaces/generated`

![](https://user-images.githubusercontent.com/1448524/155213397-2b67a16d-fb76-476e-bfcf-314903dcc046.png)

It is important to place the generated project into the `workspaces/generated` folder since this is ignored by Git in the Goldstack monorepo.

```